catalog/security/catalog-security-metacardattributeplugin/src/main/java/org/codice/ddf/catalog/security/policy/metacard/MetacardAttributeSecurityPolicyPlugin.java

```diff
@@ -36,24 +40,42 @@
  * Plugin that parses Metacard attributes for security policy information
  */
 public class MetacardAttributeSecurityPolicyPlugin implements PolicyPlugin {
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            MetacardAttributeSecurityPolicyPlugin.class);
+
+    private List<String> metacardAttributes = new ArrayList<>();
 
-    List<String> metacardAttributes = new ArrayList<>();
+    private Map<String, String> mcAttribsLookup = new HashMap<>();
 
     public List<String> getMetacardAttributes() {
         return metacardAttributes;
     }
 
     public void setMetacardAttributes(List<String> metacardAttributes) {
+        List<String> badAttributes = metacardAttributes.stream()
+                .filter(s -> !s.contains("="))
+                .collect(Collectors.toList());
+
+        if (!badAttributes.isEmpty()) {
```

Comment:
Lets not throw an error here and just take the bad attributes as they are and use the same value as key=val. Errors like this tend to just get lost when buried under config admin and then get written up as a bug.